### ISSUE TITLE
Boost Bookmarks

### ISSUE THREAD

**Issue by hurrgadev** [yacy/yacy_search_server#229]
Has anyone ever tried to boost bookmarks? Have there been any considerations? Search results of subdomains saved as bookmarks are more relevant for me. How can this best be realized?

**Comment by luccioman** [comment]
Hi @hurrgadev , personally I would also like to have such a feature. Unfortunately until now it remained in my TODO list and I didn't seriously try to dig the subject.
As far as I know, the [SearchEvent.postRanking](https://github.com/yacy/yacy_search_server/blob/db7ad7636605f9051bb7ef07bad6964cecddb163/source/net/yacy/search/query/SearchEvent.java#L1944) function could be a good candidate place to apply this kind of optional boost.

**Comment by hurrgadev** [comment]
Thanks @luccioman. Boosting works excellently there.
But the search results are not always sorted in the right order. With the Peer-to-Peer search the sorting works temporarily (remote search resorting is automated). With the private search, it's almost impossible.

**Comment by luccioman** [comment]
Good point @hurrgadev , for a proper pagination, indeed boosting bookmarked URLs should be done earlier, expressing it in the Solr query itself ideally.

So comes another idea to test :
- crawl (regularly) your own peer `/Bookmarks.html` or `/api/bookmarks/posts/all.xml` page
- then bookmarks links are in the Solr index
- in the end you may be able to craft a custom Solr boost function using this information and eventually set it up in the `/RankingSolr_p.html` page

**Comment by hurrgadev** [comment]
@luccioman At first, thanks for your great support, you are awesome.
I tried to crawl this files, but it doesnt work cause of the robots.txt. I allowed to crawl everything at my peer at ConfigRobotsTxt_p.html. No Change. I'll have to look at the code.
Next: Where can i find the RWI- and the Solr-Ranking-Algo? 
Usually the coefficients are written to sorconfig.xml. With YaCy this is different.

**Comment by luccioman** [comment]
Arr, that is possible there is an issue with robots.txt cache... You can clear it with the `/ConfigHTCache_p.html` page.

YaCy ranking coefficients are customizable in the `/RankingSolr_p.html` and `/RankingRWI_p.html` page, from the "Ranking and Heuristics" admin menu.

**Comment by hurrgadev** [comment]
@luccioman Sorry for the late answer. Are you interested in a fork pull request? I have made some changes and would like to support the project. The first pull request is that users with bookmark right are allowed to set bookmarks on yacysearch.html. Until now only admin and users with admin right were allowed to do this.

**Comment by luccioman** [comment]
Ok @hurrgadev , yes your contribution will be welcome.
